src/dsl/static.jl

```diff
@@ -256,11 +256,14 @@ function make_static_gen_function(name, args, body, return_type, annotations)
     push!(stmts, :(set_accepts_output_grad!(builder, $(QuoteNode(accepts_output_grad)))))
     bindings = Dict{Symbol,Symbol}() # map from variable name to node name
     for arg in args
+        if arg.default != nothing
+            @warn "Ignoring default value $(arg.name)=$(arg.default), unsupported in static DSL."
```

Comment:
@ztangent I think this should probably be an error. I don't see a reason to allow the user to leave the default argument in their code.

Reply:
Done!